projects/igniteui-angular/src/lib/services/exporter-common/base-export-service.ts

```diff
@@ -484,6 +660,7 @@ export abstract class IgxBaseExporter {
                 level: record.level,
                 hidden: !parentExpanded,
                 type: ExportRecordType.GroupedRecord,
+                owner: grid
```

Comment:
Didn't we agree to have owner only on the hierarchical records?

Reply:
This is fixed in my latest changes which are yet to be pushed